spec/services/migrate_statuses_spec.rb

```diff
@@ -0,0 +1,163 @@
+require 'rails_helper'
```

Comment:
If we do switch the migration to a big SQL statement, we have the upside (?) that we can delete these tests. :)